src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -190,13 +199,13 @@ const ContentHighlightStepper = ({ enterpriseId }) => {
     };
     sendEnterpriseTrackEvent(
       enterpriseId,
-      `${EVENT_NAMES.CONTENT_HIGHLIGHTS.STEPPER_STEP_CONFIRM_PUBLISH}.back.clicked`,
+      `${EVENT_NAMES.CONTENT_HIGHLIGHTS.STEPPER_STEP_CONFIRM_CONTENT}.back.clicked`,
       trackInfo,
     );
     setCurrentStep(steps[steps.indexOf(currentStep) - 1]);
   };
 
-  const closeStepper = () => {
+  const openCloseStepper = () => {
```

Comment:
nit: this function name is a bit confusing; is it opening or closing the stepper? The function's intent here is still to close the stepper, even though it opens the confirmation modal first, right? I might propose keeping the function name as something that indicates the intent to close the stepper without necessarily indicating that it's going to open the confirmation modal.